SDBLV/beliveo-files
# Add sign out

#### What does this PR do?
- Add log out(cookies)

#### Where should the reviewer start?
- beliveo-files/components/footer.php

#### How should this be manually tested? (if applies)
:one: add the footer component to an app
:two: login 
:three: verify that the cookie exists
:four: click on logout
:five: verify that the cookie is no longer
